README.md

```diff
@@ -14,56 +14,73 @@ Cute DI framework with scopes and agreeable API.
 
 ### Purpose
 
-This library is targeting to provide only an IoC-container but tries to make it really useful. 
-If you are tired of passing objects manually to create other objects which are only used to create more objects - we have a solution.
-Not all projects require an IoC-container, but check what we have.
+This library provides **IoC container** that's genuinely useful.
+If you're tired of manually passing objects just to create other objects, which then create even more — this is for
```

Comment:
imho, we lost the feeling of despair

Reply:
What do you think about these:
1. If you're exhausted by ...
2. If you're exhausted by the endless task of ...

Reply:
3. If you're exhausted from endlessly passing objects just to create other objects, only to have those objects create even more — 
3.1. this is for you
3.2. you're not alone, and we have a solution

Reply:
5. This library provides IoC container that finally makes sense.
Not every project requires IoC container, but if you're exhausted by maintaining massive dependency trees where objects exist only to instantiate other objects — we have the solution you need.

Reply:
3.2 is nice!

Reply:
Wouldn't "Not every project requires IoC container, but take a look at what we offer." be redundant in this case?

Reply:
No. The connotation here is that many projects can live without container. So there are two cases to select dishka: you really need it and you are just curious